models/src_zendesk.yml

```diff
@@ -100,12 +102,12 @@ sources:
             description: Index number of the domain name associated with the organization
 
       - name: group
+        identifier: "{%- if target.type == 'snowflake' -%} {{ var('zendesk_group_identifier', 'GROUP') }} {%- else -%} {{ var('zendesk_group_identifier', 'group') }} {%- endif -%}"
```

Comment:
Can we add this to our internal docs. I imagine this will come up in the future as well and it would be good to have a reference to why we are doing this.

Reply:
added